packages/terra-dropdown-button/src/terra-dev-site/doc/example/EmphasisDropdownButton.jsx

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-console */
+import React from 'react';
+
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import DropdownButton, { Button } from 'terra-dropdown-button/lib/DropdownButton';
+
+export default () => (
+  <DropdownButton
+    label="Dropdown"
+    variant="emphasis"
+  >
+    <Button label="1st Option" onClick={() => { console.log('hi'); }} />
```

Comment:
Let's update this example to write content to the page instead of a console.log.